daprdocs/content/en/reference/components-reference/supported-bindings/eventhubs.md

```diff
@@ -48,9 +48,14 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | connectionString | Y | Output | The [EventHubs connection string](https://docs.microsoft.com/azure/event-hubs/authorize-access-shared-access-signature). Note that this is the EventHub itself and not the EventHubs namespace. Make sure to use the child EventHub shared access policy connection string | `"Endpoint=sb://****"` |
 | consumerGroup | Y | Output | The name of an [EventHubs Consumer Group](https://docs.microsoft.com/azure/event-hubs/event-hubs-features#consumer-groups) to listen on | `"group1"` |
 | storageAccountName | Y | Output | The name of the account of the Azure Storage account to persist checkpoints data on | `"accountName"` |
-| storageAccountKey | Y | Output | The account key for the Azure Storage account to persist checkpoints data on | `"accountKey"` |
-| storageContainerName | Y | Output | The name of the container in the Azure Storage account to persist checkpoints data on | `"contianerName"` |
+| storageAccountKey | Y* | Output | The account key for the Azure Storage account to persist checkpoints data on. ***Not required if using AAD authentication.** | `"accountKey"` |
```

Comment:
only nit, have the same instructions here? 
Omit if AAD is used and AAD is preferred ?

Reply:
No this is a bit different. You *can* still choose to use a connection string for the storage account AND use AAD auth for the rest of EventHubs. That's why the message is like this.

Ultimately it depends on whether you set up your Azure credentials to give [Storage Blob Data Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#storage-blob-data-owner) role access.

You don't have to do that and can continue to use the Storage Account Connection Key if you prefer.

So this is deliberate @mukundansundar :)